### PR TITLE
fix: Adapt migration to the new product id

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

## What does this pull request change
Since product id has changed from `codeready` to `devspaces` we have to adapt migration script because names for some objects are different.

Downstream values:
```
pre-migration-prod-id-short: codeready
pre-migration-prod-deployment: codeready
pre-migration-prod-operator: codeready-operator
pre-migration-prod-checluster: codeready-workspaces
pre-migration-prod-subscription: codeready-workspaces
```

2Remove: 
```
 prod-subscription
```

## What issues does this pull request fix or reference
https://issues.redhat.com/browse/CRW-2892

## Specify the version of the product this pull request applies to
N/A

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: antora.yml [hunk 1]
```diff
@@ -96,3 +95,8 @@ asciidoc:
     project-context: che
     repo-path: ""
     theia-endpoint-image: eclipse/che-theia-endpoint-runtime:next
+    prod-migration-id-short: che
```
Comment (l0rd) [inline antora.yml]: `prod-migration-id-short` is the short ID of the product before the update right? So why not:
```suggestion
    pre-migration-prod-id-short: che
```
and the same for other variables

File: modules/administration-guide/attachments/migration/1-prepare.sh [hunk 2]
```diff
@@ -4,23 +4,24 @@ set -o nounset
 set -o pipefail
 
 K8S_CLI=${K8S_CLI:-oc}                                                           # {orch-cli}
-PRODUCT_DEPLOYMENT_NAME=${PRODUCT_DEPLOYMENT_NAME:-che}                          # {prod-deployment}
 PRODUCT_ID=${PRODUCT_ID:-eclipse-che}                                            # {prod-id}
 INSTALLATION_NAMESPACE=${INSTALLATION_NAMESPACE:-eclipse-che}                    # {prod-namespace}
-CHE_CLUSTER_CR_NAME=${CHE_CLUSTER_CR_NAME:-eclipse-che}                          # {prod-checluster}
-IDENTITY_PROVIDER_DEPLOYMENT_NAME=${IDENTITY_PROVIDER_DEPLOYMENT_NAME:-keycloak} # {identity-provider-id}
+
+MIGRATION_PRODUCT_DEPLOYMENT_NAME=${MIGRATION_PRODUCT_DEPLOYMENT_NAME:-che}      # {prod-migration-deployment}
```
Comment (l0rd) [inline modules/administration-guide/attachments/migration/1-prepare.sh]: Same remark as above. Probably this is makes things clearer:
```suggestion
PRE_MIGRATION_PRODUCT_DEPLOYMENT_NAME=${PRE_MIGRATION_PRODUCT_DEPLOYMENT_NAME:-che}      # {prod-migration-deployment}
```

File: modules/administration-guide/attachments/migration/4-wait.sh [hunk 3]
```diff
@@ -23,7 +22,7 @@ waitForComponent() {
   "${K8S_CLI}" wait --for=condition=ready pod -l app.kubernetes.io/component="${component}" -n "${namespace}" --timeout=120s
 }
 
-waitForComponent "${PRODUCT_DEPLOYMENT_NAME}" "${INSTALLATION_NAMESPACE}"
 waitForComponent "${PRODUCT_OPERATOR_NAME}" "${OPERATOR_NAMESPACE}"
+waitForComponent "${PRODUCT_DEPLOYMENT_NAME}" "${INSTALLATION_NAMESPACE}"
```
Comment (l0rd) [inline modules/administration-guide/attachments/migration/4-wait.sh:26]: Nice catch!

**Review by l0rd — APPROVED** [review]
That fixes the problem so +1. I would replace `*migration*` with `*pre-migration*` or `*post-migration*` because otherwise it's hard to follow what are the variables used for. I haven't tested though, if you have, +1 to merge it.

File: antora.yml (hunk 1 above)
Comment (tolusha) [reply antora.yml]: Agree prod-migration -> pre-migration-prod

**Comment by tolusha** [comment]
Tested succesfully.

**Review by themr0c — APPROVED** [review]

**Comment by themr0c** [comment]
Build error is legit:

   Get "https://developers.redhat.com/products/codeready-containers/overview": stopped after 10 redirects --- docs/next/administration-guide/supported-platforms/index.html --> https://developers.redhat.com/products/codeready-containers/overview

Do we have an alternate URL that actually works?
